services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -291,6 +302,27 @@ open class ApplicationService(
             else -> false
         }
 
+    private fun cancelIfSent(alluid: Int?, applicationId: Long?) {
+        if (alluid == null) {
+            logger.info {
+                "Application not sent to Allu yet, simply deleting it. id=$applicationId"
```

Comment:
These log messages are no longer related to the function. I don't think there's a reason to take this out of the delete function.

Reply:
Refactored, cancel and delete are now in the same function. Logging should make more sense now.